src/Calypso-SystemPlugins-SUnit-Browser/ClyJumpToTestClassCommand.class.st

```diff
@@ -50,9 +50,16 @@ ClyJumpToTestClassCommand >> defaultMenuItemName [
 
 { #category : #execution }
 ClyJumpToTestClassCommand >> execute [
+
+	self systemEnvironment 
+		classNamed: (self testClassNameFor: targetClass)
+		ifAbsent: [
+			(self confirm: 'Do you want to create a test class for: ' , targetClass asString )
+				ifFalse: [ ^ self ] ].
+
 	[ browser selectClass: (self testClassFor: targetClass) ]
 		on: ClyInvalidClassForTestClassGeneration
-		do: [ :ex | self inform: 'Cannot generate test class for ' , ex baseClass printString , '.' ]
+		do: [ : ex | self inform: 'Cannot generate test class for ' , ex baseClass printString , '.' ]
```

Comment:
Can you remove this space? : ex -> :ex

Reply:
I have never understood this. For method parameters it is accepted to have the argument with spaces but for block parameters it isn't?

Reply:
Let us not discuss that here. This is not because the syntax allows it that we should have strangely written method. 
99.9 % of the block are written [ :each and not [: each so let us make things simple uniform and easy.